camera_calibration/__init__.py

```diff
@@ -61,28 +61,15 @@ def calibrate(directory, rows, cols, space, win=11, save=True,
     # Setup colored output
     init()
 
-    if len(directory_out) and (directory_out[0] == '/' or
-                               directory_out[0] == '\\'):
-        directory_out = directory_out[1:]
-    if len(directory_out) and (directory_out[-1] == '/' or
-                               directory_out[-1] == '\\'):
-        directory_out = directory_out[:-1]
-
-    if len(directory) and (directory[0] == '/' or directory[0] == '\\'):
-        directory = directory[1:]
-    if len(directory) and (directory[-1] == '/' or directory[-1] == '\\'):
-        directory = directory[:-1]
-
-    print Fore.WHITE + Style.BRIGHT + Back.MAGENTA + "\nWelcome\n"
+    print(Fore.WHITE + Style.BRIGHT + Back.MAGENTA + "\nWelcome\n")
 
     # Figure out where the images are and get all file_names for that directory
-    target_directory = os.path.join(os.getcwd(), directory)
-    directory_out = os.path.join(os.getcwd(), directory_out)
-    print "Searching for images in: " + target_directory
+    target_directory = directory
+    print("Searching for images in: " + target_directory)
     file_names = os.listdir(target_directory)
-    print "Found Images:"
+    print("Found Images:")
     for name in file_names:
-        print "\t" + name
+        print("\t" + name)
```

Comment:
would be great to at some point change all of these to f-strings, but not critical